hw/ip/otbn/data/otbn.hjson

```diff
@@ -264,6 +264,8 @@
 
         Refer to the section "List of Errors" for a detailed description of the
         errors.
+
+        The value is retained until a new operation is started by writing !!CMD.
```

Comment:
I thought it persisted until the end of the next operation? I think we say that explicitly somewhere else in the documentation, so maybe it makes sense to be more exact?